lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -134,16 +143,22 @@ function dosomething_campaign_preprocess_common_vars(&$vars, &$wrapper) {
         ),
       ),
       'twitter' => array(
```

Comment:
@angaither ahh okay got it. Are social shares always displayed in STEP 1 of the campaign? Should we keep this as well and also add it to the "stuff you need" section?